detect_secrets/plugins/keyword.py

```diff
@@ -62,21 +62,29 @@
     'dummy_secret',
     'false',
     'false):',
+    'false,',
+    'false;',
```

Comment:
At some point, I wonder whether it would be better to just do:

```
FALSE_POSITIVES = []
for key in ['false', 'none', ...]:
    for suffix in [',', ';', '']:
        FALSE_POSITIVES.append(key + suffix)
```

Reply:
I agree, I think we will at some point, a lot fp's don't have a trailing char so I'm gonna hold off until it is more battle-tested.

Also been meaning to make it a `dict` instead of a tuple.